monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -306,8 +341,12 @@ object Var {
    */
   def create[A](initialValue: A)(register: Var[A] => Cancelable): Var[A] =
     new Var[A](Some(initialValue), register)
+
+
+  private[mhtml] lazy val isProduction: Boolean = Platform.checkIsProduction
```

Comment:
I would guess going through a member (and a lazy val?) might prevent stuff from being constant folded (see [this scala doc](https://github.com/scala-js/scala-js/blob/master/library/src/main/scala/scala/scalajs/LinkingInfo.scala#L16-L40)). We need check the emitted IR to see if the `isProduction` branch are properly DCE under `fullOptJS`...